### PR TITLE
feature: archive

### PR DESCRIPTION
**Pull request by nitinshr135**
- Added archive/unarchive div in Project-Settings
- Added archive project list in Organization-Settings
- Error handling in deploy-site-config
- Added error message in deploy-site-config
- Added archive-tag in project-top-card
- Big changes in project-top-card divs, optimized the div names and their CSS
- Added "state" in IProject

### CODE REVIEW

File: src/components/Sites/routes/_SharedComponent/ProjectTopCard/ProjectTopCard.tsx [hunk 1]
```diff
@@ -132,187 +132,192 @@ const ProjectTopCard = () => {
...
+                </h2>
+                <div className="archive-tag-container">
+                  {!projectLoading ? (
+                    selectedProject?.state === "ARCHIVED" ? (
+                      <span className="archive-tag">{selectedProject?.state}</span>
                     ) : (
-                      <Skeleton width={400} duration={2} />
-                    )}
-                  </p>
+                      <></>
```
Comment (rekpero) [inline src/components/Sites/routes/_SharedComponent/ProjectTopCard/ProjectTopCard.tsx:169]:
```suggestion
                      null
```
Put this if you don't have anything to show

File: src/components/Sites/routes/_SharedComponent/ProjectTopCard/ProjectTopCard.tsx [hunk 2]
```diff
@@ -132,187 +132,192 @@ const ProjectTopCard = () => {
...
+                    )
+                  ) : (
+                    <></>
```
Comment (rekpero) [inline src/components/Sites/routes/_SharedComponent/ProjectTopCard/ProjectTopCard.tsx:172]:
```suggestion
                    null
```
Same stuff

File: src/components/Dashboard/routes/Settings/Settings.tsx [hunk 3]
```diff
@@ -22,16 +23,31 @@ const Settings = () => {
...
           </div>
+          <div
+            className={`settings-bar-item ${
+              lastPath === "archive" ? "selected" : ""
+            }`}
+            onClick={(e) => history.push("/dashboard/settings/archive")}
+          >
+            Archived Projects
+          </div>
           {/* <div className="settings-bar-item">OAuth</div>
```
Comment (rekpero) [inline src/components/Dashboard/routes/Settings/Settings.tsx:34]: Remove these

File: src/components/Dashboard/routes/Settings/routes/SettingsArchive/SettingsArchive.scss [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+                  padding: 1px 10px;
+                  width: fit-content;
+                  background-color: #303030;
+                  color: var(--white);
+                  border-radius: 12px;
+                  margin-top: 0.5rem;
+                }
+              }
+            }
+            // .protocol-tag-container {
```
Comment (rekpero) [inline src/components/Dashboard/routes/Settings/routes/SettingsArchive/SettingsArchive.scss:120]: Remove these if not required later

**Review by rekpero — CHANGES_REQUESTED** [review]
LGTM @nitinshr135 !!
Just some small update, please resolve them

File: src/components/Sites/routes/_SharedComponent/ProjectTopCard/ProjectTopCard.tsx (hunk 2 above)
Comment (nitinshr135) [reply src/components/Sites/routes/_SharedComponent/ProjectTopCard/ProjectTopCard.tsx]: committed the requested changes

**Review by rekpero — APPROVED** [review]

**Pull request merged**